lib/tizen_plugins.dart

```diff
@@ -629,3 +633,35 @@ Future<void> renderTemplateToFile(
   await file.create(recursive: true);
   await file.writeAsString(renderedTemplate);
 }
+
+/// See: [_writeFlutterPluginsList] in flutter_plugins.dart
+Future<void> _writeTizenPluginsExtraInfo(
+  FlutterProject project,
+) async {
+  final File extraInfoFile =
+      project.directory.childFile('.flutter-tizen-plugins-extra-info');
+
+  final List<TizenPlugin> plugins = await findTizenPlugins(project);
+  final List<Map<String, Object>> pluginInfo = <Map<String, Object>>[];
+  for (final TizenPlugin plugin in plugins) {
+    pluginInfo.add(<String, Object>{
+      'name': plugin.name,
+    });
+  }
+
+  final Map<String, Object?> result = <String, Object>{};
+  result['info'] =
+      'This is a generated file; do not edit or check into version control.';
+  result['plugins'] = pluginInfo;
+  result['date_created'] = globals.systemClock.now().toString();
+  result['dart_version'] = globals.flutterVersion.dartSdkVersion;
+  result['flutter_version'] = globals.flutterVersion.frameworkVersion;
+  result['engine_version'] =
+      globals.cache.getStampFor(kTizenEngineStampName) ?? '';
+  result['embedder_version'] =
```

Comment:
It looks better without `_version`.
'dart', 'flutter', 'engine', 'embedder', 'flutter-tizen' <- hash